### DISCUSSION TITLE
Upload script

### DISCUSSION THREAD

**Discussion by AlinB87** [community/community#32300] [Other Feature Feedback, Questions, & Ideas]
How can i upload a script [GitHub - jojerd/UnifiedContentCorrector: Exchange 2013 Exchange 2016 and Exchange 2019 fix Unified Content folder](https://github.com/jojerd/UnifiedContentCorrector) on [Microsoft - CSS-Exchange](https://microsoft.github.io/CSS-Exchange/#popular-scripts)

**Comment by airtower-luna** [comment]
Go to the repository behind that site: https://github.com/microsoft/CSS-Exchange Check the contributing guidelines there. :slightly_smiling_face:
